### PR TITLE
Feat/create packages with classes

### PR DESCRIPTION
**Pull request by zonglun99**
Created new packages: commands, data(exception and ingredient) , parser, storage and ui.
Created new classes: Command, FridgetException, Parser, Ui, Ingredient, IngredientList and Storage.

### CODE REVIEW

File: src/main/java/seedu/storage/IngredientList.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package seedu.storage;
+
+import seedu.data.ingredient.Ingredient;
+import java.util.ArrayList;
+
+public class IngredientList {
+    protected ArrayList<Ingredient> ingredientList;
+
+    public IngredientList(ArrayList<Ingredient> ingredientList) {
+        ingredientList = new ArrayList<>(100);
```
Comment (uosjapuelks) [inline src/main/java/seedu/storage/IngredientList.java:10]: I think we don't have to specify the (100) but I think can just merge first and it should be:

ingredientsList = new ArrayList<Ingredient>();

**Review by uosjapuelks — COMMENTED** [review]
Apart from ingredientsList, everything else seems fine.

File: src/main/java/seedu/storage/IngredientList.java (hunk 1 above)
Comment (zonglun99) [reply src/main/java/seedu/storage/IngredientList.java:10]: okay we can update that later

**Review by alvynben — APPROVED** [review]
LGTM!

**Pull request merged**